samples/publishWithRetrySettings.js

```diff
@@ -50,7 +50,7 @@ function main(
   });
 
   async function publishWithRetrySettings() {
-    const formattedTopic = publisherClient.topicPath(projectId, topicName);
+    const formattedTopic = publisherClient.projectTopicPath(projectId, topicName);
```

Comment:
I am worried this breaking change will get lost in the shuffle for folks.  Can we make this super clear in the breaking changes section of the change log?

Reply:
Make sense.  But not very clear here, should I update change log in this PR?

Reply:
@bcoe what's the recommended way to do this?  I've done longer form "before / after" descriptions for breaking changes that were meaningful in the past.  Is the process here to manually update the CHANGELOG in a followup PR?